src/igvfd/schemas/changelogs/biosample.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog for *`biosample.json`*
 
+### Release v9
+
+* Add `modification`.
 
 ### Minor changes since Release v7
```

Comment:
I don't think we need a "minor changes since..." in the changelog for abstract classes but I didn't want to modify that in this ticket.

Reply:
agree